module/BingoMotion/BlockThrower.cpp

```diff
@@ -10,18 +10,119 @@ BlockThrower::BlockThrower() : BingoMotion(1, 1) {}
 
 void BlockThrower::setBlockThrow(bool isClockwise)
 {
-  double targetDistance = 105;
-  int runPwm = 20;
+  double runDistance = 105;
+  int runPwm = 30;
   int angle = 45;
-  int rotatePwm = 20;
+  int rotatePwm = 100;
+  int leftSign;
+  int rightSign;
+  double diffLeftDistance;
+  double diffRightDistance;
+  int leftPwm;
+  int rightPwm;
+  double targetDistance = M_PI * TREAD * angle / 360;
+  double targetLeftDistance;
+  double targetRightDistance;
+  int armPwm = 90;
 
   //黒線の奥まで直進する
-  straightRunner.runStraightToDistance(targetDistance, runPwm);
+  straightRunner.runStraightToDistance(runDistance, runPwm);
+  // アームを水平にする処理を無効化
+  ArmMotion::setKeepFlag(false);
   //回頭する
   if(isClockwise) {
-    rotation.rotateRight(angle, rotatePwm);
+    leftSign = 1;
+    rightSign = -1;
```

Comment:
```suggestion
    leftSign = isClockwise ? 1 : -1;
    rightSign = isClockwise ? -1 : 1;
```
とかじゃダメなの？時計回りも反時計回りも同じ処理に見えるけど，if文で分けてるのには理由があるの？
異なった調整をしてる，もしくはしたいとか？